### ISSUE TITLE
Fix tycho source feature problems

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext#2083]
https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md#tycho-source-feature-generation-moved-from-tycho-extras-to-tycho-core
https://github.com/eclipse/tycho/issues/1216

**Comment by cdietrich** [comment]
```
					<execution>
						<id>feature-source</id>
						<goals>
						    <goal>feature-source</goal>
						</goals>
					</execution>
```

**Comment by cdietrich** [comment]
fixed in 2.28
